src/eth/state.yaml

```diff
@@ -84,3 +84,18 @@
     name: Account
     schema:
       $ref: '#/components/schemas/AccountProof'
+- name: eth_getAccount
+  summary: Returns the account at given address, or 'null' if the account does not exist in the state.
+  params:
+    - name: Address
+      required: true
+      schema:
+        $ref: '#/components/schemas/address'
+    - name: Block
+      required: false
+      schema:
+        $ref: '#/components/schemas/BlockNumberOrTag'
+  result:
+    name: Account
+    schema:
```

Comment:
I think you need to make this a `oneOf` and add `null` as a possibility. You can find an example here: https://github.com/ethereum/execution-apis/blob/4015a8c6629a92f4d56a81edad1501b82e1edd7a/src/schemas/receipt.yaml#L78-L84